### PR TITLE
Add color-operations package

### PR DESCRIPTION
**Pull request by giswqs**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

https://github.com/conda-forge/rio-tiler-feedstock/pull/27

### CODE REVIEW

**Comment by jsignell** [comment]
I am happy to be added as a maintainer if you like @giswqs

**Comment by giswqs** [comment]
@jsignell Happy to help.

File: recipes/color-operations/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python >=3.8
+    - numpy
+    - cython
+    - pip
+  run:
+    - python >=3.8
+    - {{ pin_compatible('numpy') }}
```
Comment (jsignell) [inline recipes/color-operations/meta.yaml:27]: I wonder if you need cython in the run as well
```suggestion
    - {{ pin_compatible('numpy') }}
    - cython
```

File: recipes/color-operations/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+test:
+  imports:
+    - color_operations
```
Comment (jsignell) [inline recipes/color-operations/meta.yaml:31]: you might need to explicitly list the modules 
```suggestion
    - color_operations.colorspace
```

File: recipes/color-operations/meta.yaml (hunk 1 above)
Comment (jsignell) [reply recipes/color-operations/meta.yaml:31]: no that's not right :facepalm:

**Comment by giswqs** [comment]
Got some hints from [here](https://conda-forge.org/docs/maintainer/knowledge_base.html#cross-compilation). Let's see if the builds will pass this time.

File: recipes/color-operations/meta.yaml (hunk 2 above)
Comment (jsignell) [reply recipes/color-operations/meta.yaml:36]: I just noticed that they do that in the dask feedstock :shrug:

**Comment by vincentsarago** [comment]
FYI a new version (0.1.1) of `color-operations` should be available soon (wheels takes 1h to build 😓)

File: recipes/color-operations/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+{% set name = "color-operations" %}
+{% set version = "0.1.0" %}
```
Comment (vincentsarago) [inline recipes/color-operations/meta.yaml:2]:
```suggestion
{% set version = "0.1.1" %}
```

let's use the newest version

**Comment by vincentsarago** [comment]
@giswqs `color-operations` is a fork of rio-color so we should have a look at https://github.com/conda-forge/rio-color-feedstock to see if we're missing something

**Comment by giswqs** [comment]
@vincentsarago Still the same error. It might be related to how `color_operations.colorspace` is handled in setup.py.
https://github.com/vincentsarago/color-operations/blob/main/setup.py#L50

File: recipes/color-operations/meta.yaml (hunk 2 above)
Comment (jsignell) [reply recipes/color-operations/meta.yaml:36]: Nah that is probably not useful

**Comment by giswqs** [comment]
@jsignell Nice! Thank you very much for fixing it. I can close this issue now.

**Pull request closed without merging**